src/parser.rs

```diff
@@ -6912,40 +6912,6 @@ mod tests {
         });
     }
 
-    #[test]
-    fn test_parse_limit() {
-        let sql = "SELECT * FROM user LIMIT 1";
-        all_dialects().run_parser_method(sql, |parser| {
-            let ast = parser.parse_query().unwrap();
-            assert_eq!(ast.to_string(), sql.to_string());
-        });
-
-        let sql = "SELECT * FROM user LIMIT $1 OFFSET $2";
```

Comment:
Did we lose these tests? I see a mysql dialect test for offset but not one that touches the other dialects. Apologies if I missed it!

Reply:
Good spot. I should have called this out explicitly -- this test was replicated and therefore redundant. 

The pre-existing test in sqlparser_common.rs is almost identical: 

https://github.com/sqlparser-rs/sqlparser-rs/blob/784a19138f269dab4ea3e1ad3c9da565d20c9155/tests/sqlparser_common.rs#L6092-L6104